src/components/Chart.js

```diff
@@ -1,5 +1,64 @@
-import React from "react";
+import React, { Component } from "react";
+import ChartComponent, { Line } from "react-chartjs-2";
 
-const Chart = () => <div>Chart Component</div>;
+let values = [
+  {
+    date: "some_date_1",
```

Comment:
Date should use this object:
https://developer.mozilla.org/pl/docs/Web/JavaScript/Referencje/Obiekty/Date